dhcpv4/dhcpv4.go

```diff
@@ -468,6 +469,19 @@ func (d *DHCPv4) ToBytes() []byte {
 	// Write all options.
 	d.Options.Marshal(buf)
 
+	// DHCP is based on BOOTP, and BOOTP messages have a minimum length of
+	// 300 bytes per RFC 951. This not stated explicitly, but if you sum up
+	// all the bytes in the message layout, you'll get 300 bytes.
+	//
+	// Some DHCP servers and relay agents care about this BOOTP legacy B.S.
+	// and "conveniently" drop messages that are less than 300 bytes long.
+	//
+	// We subtract one byte for the OptionEnd option.
+	const bootpMinLen = 300 - 1
+	if buf.Len() < bootpMinLen {
```

Comment:
nit: what about subtracting the size of OptionEnd here and below, instead? I'd rather leave `bootpMinLen` semantically correct (i.e. 300) even if it's unexported and clearly commented

Reply:
done